README.md

```diff
@@ -427,3 +459,94 @@ You are busy on this interval!
 
 You have an event from 2017-09-21T21:00:00+08:00 to 2017-09-21T22:00:00+08:00
 ```
+If you are not free on that time span, for example you have 2 classes on that time span:
+```
+10:30 - 11:30 CZ1003 LEC
+14:30 - 16:30 CZ1004 LAB
+```
+Hence, the bot will return:
+```
+You have 2 events occuring between 2017-10-09 08:00 and 2017-10-09 16:00
+
+1. 2017-10-09 10:30 until 2017-10-09 11:30
+2. 2017-10-09 14:30 until 2017-10-09 16:00
+
+What you probably want to do next:
+
+Run /isfree again with different datetime
+```
+
+### 4.8 See All Your Upcoming Events
+
+You can view your upcoming events through the **/getupcomingevents** command. The bot will respond in the following manner:
+
+```
+Please enter how many upcoming events are you looking for!
+
+Press!
+```
+It will then display a markup keyboard displaying a number from 1 to 7. You can choose to display upto 7 upcoming events. 
+In the case where you clicked **7**, the bot will respond in this following manner:
+```
+Getting 7 upcoming event(s) for you
+Here they are!
+```
+And it will display the first seven of your upcoming events (including your courses).
+It displays the **name of the event, date of the event and time interval of the event**.
+
+### 4.9 To See All The Commands You Have In Your Bot
+
+To see all the commands available in your bot, you may type in **/help** and the bot will display the commands available in the bot in the following manner:
+```
+Basic Commands 
+/start - Send welcome message 
+/help - list available commands 
+/quit - Send good bye message 
+
+General Commands 
+/isfree - To check whether you are free at a certain time interval 
+/getupcomingevent - List your upcoming events 
+
+Event-related Commands 
+/addevent - Add an event to your Google Calendar 
+/removeevent - Remove an event from your Google Calendar 
+/getevent - List all events that you have added 
+
+Course-related Commands 
+/addcourse - Add a course schedule to your Google Calendar 
+/removecourse - Remove a course schedule from your Google Calendar 
+/getcourse - List all courses that you have added 
+/setstudenttype - Set your student type (Full Time or Part Time) 
+/addfirstweek - Add the first weekday, i.e. Monday, of your first week and recess week
+```
+
+## Chapter 5: Further Information
```

Comment:
Yup, I agree to put this information in a separate section, otherwise the readme files would contain too much information in a chapter. That being said, it would not be **read**me - able.